test/functional/p2p_invalid_messages.py

```diff
@@ -66,7 +66,7 @@ def run_test(self):
         msg_at_size = msg_unrecognized("b" * valid_data_limit)
         assert len(msg_at_size.serialize()) == msg_limit
 
-        with node.assert_memory_usage_stable(perc_increase_allowed=0.03):
+        with node.assert_memory_usage_stable(perc_increase_allowed=0.5):
```

Comment:
Doesn't percent mean it is denominated in hundreds? So maybe rename it at the same time to `increase_allowed`? And maybe require it to be a named arg with `, *, increase_allowed` in the signature?

Reply:
Good suggestions; added in a subsequent commit.